.storybook/preview.js

```diff
@@ -78,8 +79,15 @@ const withTrackingAction = (Story) => (
   </TrackingRoot>
 );
 
+addParameters({
+  performance: {
+    allowedGroups: ['client'],
```

Comment:
Since the SSR tests work locally, I'd keep them enabled there:
```suggestion
    allowedGroups: process.env.NODE_ENV === 'production' ? ['client'] : ['server', 'client'],
```

Reply:
Done in ba968e2